packages/components/src/components/form/Checkbox/index.tsx

```diff
@@ -67,7 +67,7 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
 type IconWrapperProps = Omit<Props, 'onClick'>;
 
 const handleKeyboard = (event: React.KeyboardEvent<HTMLElement>, onClick: Props['onClick']) => {
-    if (event.keyCode === 32) {
+    if (Number(event.code) === 32) {
```

Comment:
hey, wouldnt it be better to avoid conversion to number simply by doing
```
event.code === '32'
```

But most importantly. I don't thing this will work. event.code does not hold the same values like event.keyCode does. 

You can check for example here https://keycode.info/